### PR TITLE
update hosting doc with updateURL info

### PR DESCRIPTION
**Pull request by ghost**
Thanks for filing that issue on tab split.  I'm not sure if that's documented anywhere, but this seems like a good place.

### CODE REVIEW

File: docs/development/hosting.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ Your repository must include a `npm run package` script which, when run, will
 build the add-on and leave it in a file named `addon.xpi` in the root
 directory.
 
+Your add-on must use a custom update URL in your `install.rdf` file to make
+updates work.  An example follows (replace `%ID%` with your add-on's ID):
+```xml
+<em:updateURL>https://testpilot.firefox.com/files/%ID%/updates.json</em:updateURL>
+```
+
```
Comment (pdehaan) [inline docs/development/hosting.md]: Do we want an example of manifest.json `update_url`, a la Notes? 

```js
  "applications": {
    "gecko": {
      "id": "notes@mozilla.com",
      "strict_min_version": "57.0",
      "update_url": "https://testpilot.firefox.com/files/notes@mozilla.com/updates.json"
    }
  },
```
&mdash; via https://github.com/mozilla/notes/blob/191440a78c52ebc409fc01a9fa61c2bfe557d726/src/manifest.json#L8-L14

Comment (pdehaan) [reply docs/development/hosting.md]: And purely for my own amusement/reference, here's the `applications.gecko.update_url` and `<em:updateURL>` for the other current experiments:

- [**Min-Vid's** /install.rdf](https://github.com/meandavejustice/min-vid/blob/574252f8f64b96768a3318b2d1a9b6ed977d3d71/install.rdf#L23)
- [**SnoozeTabs'** /package.json](https://github.com/bwinton/SnoozeTabs/blob/a005e7fd5be863492790bef05ad6a2128e6488f1/package.json#L81)
- [**SpeakToMe's** /extension/manifest.json](https://github.com/mozilla/speaktome/blob/813867933c21b8e29940747c3bc628e1750e23a8/extension/manifest.json#L17)

Comment (ghost) [reply docs/development/hosting.md]: yeah, that's probably a better example than install.rdf anyway.  I'll update to include both

**Comment by pdehaan** [comment]
:+1: LGTM. Not sure if @meandavejustice or @chuckharmston wants to give a final 👀 before merging.

**Pull request merged**